islandora.module

```diff
@@ -545,12 +545,13 @@ function islandora_entity_view(array &$build, EntityInterface $entity, EntityVie
   $route_match_item = \Drupal::routeMatch()->getParameters()->get($entity->getEntityTypeId());
   // Ensure the entity matches the route.
   if ($entity === $route_match_item) {
-    if ($display->getComponent('field_gemini_uri')) {
+    $flysystem_config = Settings::get('flysystem');
+    $fedora_root = $flysystem_config['fedora']['config']['root'];
```

Comment:
This attempted access would likely start throwing warnings in environments where it isn't defined.